### ISSUE TITLE
Update license in twisted.spec

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3505]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3505|
|Type|defect|
|Created|2008-10-27 00:32:35Z|

The [admin/twisted.spec](admin/twisted.spec) still refers to the LGPL, let's change this to MIT.

Attachments:

* [spec.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b18/b18a8b28a8c0453af82f0ab4ee9e7ce31cbf11ac/a2b27ae7bb15c0c59b8b3535442af54361f410db.patch) (393 bytes) - added by thijs on 2008-10-27 00:34:05Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3505 3505
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1225067555000000 1225067555000000
changetime__1225135965000000 1225135965000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Great, please apply.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [25139]) Apply spec.patch: Update license type (MIT instead of LGPL).

Author: thijs
Reviewer: exarkun
Fixes: [#3505](https://github.com/twisted/twisted/issues/3505)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
